sei_dbt/README.md

```diff
@@ -1,4 +1,181 @@
-Welcome to your new dbt project!
+## Overview of dbt-duckdb
+
+This project is designed to load multiple CSV files into a DuckDB database using **dbt** (Data Build Tool). It helps automate the process of transforming, modeling, and querying data from CSV files in a simple and efficient way. DuckDB is used as the local database engine, and dbt orchestrates the loading and transformation process.
+
+
+
+## Features
+
+
+=======
+- **Automated CSV Loading:** Easily load multiple CSV files into DuckDB.
+- **Data Transformation:** Use dbt to perform SQL-based data transformations on the loaded CSVs.
+- **Lightweight Database:** Leverages DuckDB, which is optimized for efficient data processing on local files.
+- **Flexible:** Customize dbt models to suit your business logic and transformation requirements.
+
+---
+
+## Requirements
+
+Since we use `dbt-duckdb` Python package, before starting, ensure you have the following tools installed:
+
+
+1. **Python** (>= 3.7)
+2. **dbt** (>= 0.19.0)
+3. **DuckDB** (Python package or standalone)
+4. **CSV Files:** Ensure you have your source CSV files ready.
+
+
+To install the required Python packages:
+
+```bash
+pip install dbt-duckdb duckdb
+```
+
+More instructions are available in https://github.com/mehd-io/dbt-duckdb-tutorial/
+
+---
+
+## Setup
+
+Follow these steps to set up and configure the project:
+
+1. **Clone the repository:**
+
+   ```bash
+   git clone <your-repository-url>
+   cd <project-folder>
+   ```
+
+2. **Initiate a new projec:**
+
+If you start a new project, 
+
+```bash
+dbt init
+```
+It will:
+
+    ask you to name your project
+    ask you which database adapter you're using (or to Supported Data Platforms)
+    prompt you for each piece of information that dbt needs to connect to that database: things like account, user, password, etc
+
+Then, it will:
+
+    Create a new folder with your project name and sample files, enough to get you started with dbt
+    Create a connection profile on your local machine. The default location is ~/.dbt/profiles.yml.
+
+
+3. **Configure dbt profiles:**
+
+   dbt uses a `profiles.yml` file to connect to DuckDB. Ensure that your `profiles.yml` file is correctly configured as follows:
```

Comment:
does this profile file have to be pushed to github? can it live locally only, and included in gitignore? (I'm not sure if it's actually needed on github or not, I am naively learning about dbt here)

Reply:
This is a great question. I researched it and it turns out that the profile file could be used to save credentials and it should be git ignored. The profile file could be global instead of project-specific. I will do more research and come back this point. The safepath problem could be solved using the profile file.

Reply:
Yes. Now `profiles.yml` is located in user home directory.